build/_k-standard-goals.shade

```diff
@@ -187,6 +189,18 @@ default Quiet='${ false }'
     }
   }
 
+#nuget-verify target='test' if='File.Exists(PACKAGELIST_JSON_FILENAME)' description='Verify if all the packages are generated properly'
+  exec program='cmd' commandline='/C dnu commands install --source ${DNX_TOOLS_FEED} NugetPackageVerifier --packages ${BASE_DIR}\commands\' if='ShouldVerifyNupkgs'
```

Comment:
I'm a little concerned about this. The tool will run using the active DNX. We don't have a way of "pinning" it to a particular DNX. There is a chance that future DNX changes will break it.